k8s/yaml/README.md

```diff
@@ -0,0 +1,28 @@
+# K8s example
+
+This provides an extremely basic "bare-bones" GKE deployment example for K8s using:
+ - BoltDB and PD-SSD for persistence
+ - Basic `ClusterIP` service for exposing the web ui
+
+## Quickstart
+
+1) Edit the configuration in `config_map.yaml`
+
+2) Add the service account key secret in secret.yaml by parsing an account key using `base64`, e.g.:
```

Comment:
It may be worthwhile to have a brief comment that this is only required if not using workload identity in GKE.

Reply:
Done.